source/gui/_addonStoreGui/controls/details.py

```diff
@@ -314,6 +314,13 @@ def _refresh(self):
 						details.sourceURL
 					)
 
+					if details.reviewUrl is not None:

```

Comment:
Shouldn't the "reviewUrl" variable in all of these places, be capitalized the same as "details.sourceURL"?